config/routes.rb

```diff
@@ -20,7 +20,7 @@
 
   resources :items, only: [:new, :show]
   resources :users, only: [:show, :create]
-  # resources :admins
-
+  resources :orders, only: [:new, :create, :show]
```

Comment:
I think one of the main benefits of nesting is having access to the `has_many` part of the relationship in params coming from the routes. That's not really the case with user (assume you're using `current_user` in these cases). Let's sleep on it and discuss tomorrow.